### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by Magalame** [symengine/symengine.hs#16]
It seems like there hasn't been a commit in one year, does that mean the project is dead?

**Comment by isuruf** [comment]
Yes, it is. If anyone wants to revive it they are welcome to and I can give access.

**Comment by certik** [comment]
@Magalame yes, as @isuruf said, right now we don't have anybody to maintain the Haskell wrappers. But we maintain the core C++ library regularly, and we have all these wrappers here so that if somebody becomes interested, they can just jump in and help out. We will be happy to give merge access to anybody who starts contributing.
